cilium/cmd/cmdref.go

```diff
@@ -0,0 +1,43 @@
+// Copyright 2017 Authors of Cilium
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"github.com/spf13/cobra"
+	"github.com/spf13/cobra/doc"
+)
+
+var dir string
+
+var cmdRef = &cobra.Command{
+	Use:   "cmdref",
+	Short: "Generate Cilium command reference",
+	Run: func(cmd *cobra.Command, args []string) {
```

Comment:
Set `Hidden: true`?
https://godoc.org/github.com/spf13/cobra#Command

Reply:
Cool was not aware of that. Makes sense to hide it.